dev/config_conjur_identity/configure-conjur-identity/playbook.yml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
I think the `dev/config_conjur_identity` dir is unnecessary.
Maybe we could include a `dev/playbooks` directory and relocate the playbooks:
```
dev/
  playbooks/
    playbook.yml (maybe given a more descriptive name)
    inventory_playbook.yml
```

Reply:
I have changed the path and name of playbook.yml , we need iteration in next Jira ticket that is why I have not removed iteration from start.sh file .